bin/nyc.js

```diff
@@ -46,6 +46,16 @@ var yargs = require('yargs/yargs')(process.argv.slice(2))
     type: 'boolean',
     describe: 'whether or not to instrument all files of the project (not just the ones touched by your test suite)'
   })
+  .option('exclude', {
+    alias: 'ex',
+    default: [],
+    describe: 'a list of specific files and directories that should be excluded from coverage, glob patterns are supported, node_modules is always excluded'
+  })
+  .option('include', {
+    alias: 'in',
+    default: ['**'],
```

Comment:
`include` is used in [this line](https://github.com/bcoe/nyc/blob/3403ca17991fded623a05fda3e3398375320e1d9/index.js#L191):

```js
return (!this.include || micromatch.any(relFile, this.include)) && !micromatch.any(relFile, this.exclude)
```

I think with this change, when `nyc` is used through the CLI, it'll always call `micromatch.any`. Without a default it would just short-circuit the logic. We should definitely have the default in the documentation but maybe we should also [set in the constructor](https://github.com/bcoe/nyc/blob/3403ca17991fded623a05fda3e3398375320e1d9/index.js#L39:L42) so we can remove that logic, *or* set `this.include` to `false` if it's just `**`, so we can avoid the unnecessary match.

The difference between the CLI codepath and the embedding code path bothers me a little :smile: